docs/db/migrations.md

```diff
@@ -46,7 +46,6 @@ migrateDB(dbConfig, schemaText, {
   - `scalarMap`: Custom scalar mapping. Default: `null`.
   - `mapListToJson`: Map scalar lists to JSON column type by default.
   - `debug`: display debugging information and SQL queries.
-  - `removeDirectivesFromSchema`: removes all directives from the GraphQL schema.
```

Comment:
@craicoverflow FYI we have killed this as it was just leftover from using CRUD markers as directives